src/assets/styles/commons/Footer.scss

```diff
@@ -0,0 +1,114 @@
+body {
+	margin: 0;
+	font-family: Helvetica, monospace;
+}
+
+.footer-container {
+	display: flex;
+	flex-direction: row;
+	height: 535px;
+	width: 1920px;
```

Comment:
We are using a lot of absolute CSS units, Do we have any dream of making the site responsive?